src/psyclone/psyGen.py

```diff
@@ -255,6 +261,7 @@ class PSy(object):
 
     '''
     def __init__(self, invoke_info):
+        # pylint:disable=dangerous-default-value
```

Comment:
Unless you have a good reason for disabling this, I would prefer to leave it alone (or fix the dangerous default value :-))